alfresco-health-processor-platform/src/test/java/eu/xenit/alfresco/healthprocessor/reporter/InMemoryHealthReportsStore.java

```diff
@@ -0,0 +1,44 @@
+package eu.xenit.alfresco.healthprocessor.reporter;
+
+import eu.xenit.alfresco.healthprocessor.plugins.api.HealthProcessorPlugin;
+import eu.xenit.alfresco.healthprocessor.reporter.api.NodeHealthReport;
+import eu.xenit.alfresco.healthprocessor.reporter.api.NodeHealthStatus;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class InMemoryHealthReportsStore implements HealthReportsStore {
```

Comment:
This is not used anywhere. Will we use it in the future?

Reply:
This was the original implementation. Could still become useful for testing later.